### PR TITLE
integration tests: skip shared ROS test on non-xenial

### PR DESCRIPTION
**Pull request by kyrofa**
- [x] Have you followed the [guidelines for contributing](https://github.com/snapcore/snapcraft/blob/master/CONTRIBUTING.md)?
- [x] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?
- [x] If this is a bugfix. Have you checked that there is a bug report open for the issue you are trying to fix on [bug reports](https://bugs.launchpad.net/snapcraft)?
- [x] If this is a new feature. Have you discussed the design on the [forum](https://forum.snapcraft.io)?
- [x] Have you successfully run `./runtests.sh static`?
- [x] Have you successfully run `./runtests.sh unit`?

### CODE REVIEW

File: integration_tests/test_catkin_shared_ros.py [hunk 1]
```diff
@@ -20,9 +20,12 @@
 
 import integration_tests
 
+from snapcraft.tests import skip
```
Comment (sergiusens) [inline integration_tests/test_catkin_shared_ros.py]: this is the wrong import!

Comment (sergiusens) [reply integration_tests/test_catkin_shared_ros.py]: ah, pyramid is in the `integrationtests` `contrel` file. All good. Still faster to use the leaner version :wink:

Comment (kyrofa) [reply integration_tests/test_catkin_shared_ros.py]: Ugh, you scared me. But yeah, as long as the tests pass, we'll clean that up soon.

**Comment by elopio** [comment]
aaaand it failed.
But because ruby timed out. This one was successfully skipped:
```
test_shared_ros_builds_without_catkin_in_underlay (test_catkin_shared_ros.CatkinSharedRosTestCase)
test_catkin_shared_ros.CatkinSharedRosTestCase.test_shared_ros_builds_without_catkin_in_underlay ... skipped 'ROS Kinetic only targets Xenial'
```
So, merge! And one more try with the changelog.

**Pull request merged**